Cargo.toml

```diff
@@ -36,14 +37,14 @@ clap = "=4.4.17"
 clap_complete = "=4.4.7"
 reqwest = "0.11.20"
 
-# https://github.com/denoland/deno_ast/blob/main/Cargo.toml
-swc_atoms = "=0.6.5"
-swc_common = "=0.33.17"
-swc_ecma_ast = "=0.110.17"
-swc_ecma_codegen = "=0.146.54"
-swc_ecma_parser = "=0.141.37"
-swc_ecma_transforms_typescript = "=0.186.17"
-swc_ecma_visit = "=0.96.17"
+# update from https://github.com/denoland/deno_ast/blob/main/Cargo.toml
+swc_atoms = "=0.6.7"
+swc_common = "=0.33.26"
+swc_ecma_ast = "=0.113.4"
+swc_ecma_codegen = "=0.149.1"
+swc_ecma_parser = "=0.144.1"
+swc_ecma_transforms_typescript = "=0.189.1"
+swc_ecma_visit = "=0.99.1"
```

Comment:
These are only related to the typescript codegen right?
I think we talked about removing these along with the current typescript codegen at some point as these are going to be replaced by the full mdk rewrite anyways.

Reply:
Handled in #739 .